src/config/env.json

```diff
@@ -29,14 +30,17 @@
   },
   "qa": {
     "DD_ENV": "protocol-qa",
-    "DD_TAGS": "env:protocol-qa"
```

Comment:
I think we also want twillio enabled in dev (we use a dev api key so that twillio accepts it but doesn't actually send anything)